### PR TITLE
Move to anexia github organisation

### PR DESCRIPTION
**Pull request by anx-mfischer**
Happy Hacktoberfest

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.1] - 2022-10-21
```
Comment (beachmachine) [inline CHANGELOG.md]: As this package has no release yet, I'd suggest to just put the changes under "Unreleased".

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+### Added
+- This CHANGELOG file
+
+### Changed
+- Moved from https://github.com/anexia-it/django-aws-iot to https://github.com/anexia/django-aws-iot
```
Comment (beachmachine) [inline CHANGELOG.md]: We do not mention the move between organisations if not needed. However, we may want to add "Initial release" under "Added".

**Review by beachmachine — CHANGES_REQUESTED** [review]
Also `setup.py` must be updated to reflect the new URL.

**Comment by anx-mfischer** [comment]
Requested changes are done.

**Review by beachmachine — APPROVED** [review]

**Pull request merged**